1080
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 844, in <module>
    main()
  File "run_linearization_heuristic.py", line 100, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-08n46: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=12852867.0
